src/test/java/org/folio/service/mapping/MappingServiceUnitTest.java

```diff
@@ -15,21 +22,41 @@
 @RunWith(MockitoJUnitRunner.class)
 public class MappingServiceUnitTest {
 
+  @Mock
+  private MappingSettingsProvider mappingSettingsProvider;
+  private String jobExecutionId = "67429e0e-601a-423b-9a29-dec4a30c8534";
+  private OkapiConnectionParams params = new OkapiConnectionParams();
+
+  @Before
+  public void mockSettings() {
+    Settings settings = new Settings();
```

Comment:
Why can't you just use the existing mock file. This is redundant isn't it, and any change needs to be made it multiple places.